spec/graphql/mutations/sign_in_musician_spec.rb

```diff
@@ -23,4 +23,28 @@ def query
   }
   GQL
   end
-end
\ No newline at end of file
+
+  it 'errors on failure to authenticate' do
+    musician_2 = Musician.create!(email: 'bruce@mail.com', password: 'password', name: 'Phil Brazil', photo: 'www.photo.com', phone: '5597995111')
+    post '/graphql', params: { query: query2 }
+    response_body = JSON.parse(response.body, symbolize_names: true)
+    
+    expect(response_body[:data][:signInMusician]).to eq(nil)
+    expect(response_body[:errors][0][:message]).to eq('Invalid login credentials')
```

Comment:
I would expect a status code too.

Reply:
GraphQL only ever responds with a 200 Status, but we checked the docs and there's an "extension" key we added with a :code that is now "UNAUTHORIZED".